docs/upgrade.html

```diff
@@ -26,6 +26,10 @@ <h5><a id="upgrade_400_notable" href="#upgrade_400_notable">Notable changes in 4
             <ul>
                 <li><b>Common</b>
                     <ul>
+                        <li>
```

Comment:
Could you please move this to the 'Other Changes' section? This section seems to be focused on configuration changes.